javascript/javascript1/week3/script.js

```diff
@@ -74,4 +74,7 @@ return 'Hours of :'+hoursFirstPart+ ' and minutes of :'+hoursSecondPart;
 }
 
 
-console.log(travelTime); // 8 hours and 38 minutes
```

Comment:
I see that your hw is already merged to master. So I will give you some feedback based on what you have on the master branch of the js week3 hw using the line numbers of the script.js file:
- l.18 -> A better way to find the index of a value in an array instead of a for loop is to use ```index names.indexOf(nameToRemove);```
- l.66 -> Avoid using var, instead use const in this specific case cause you will not change the value of the variable.